src/lessons/06_introduction_to_javascript/introduction-to-javascript.slides.html

```diff
@@ -0,0 +1,1147 @@
+
+            <pre><code data-trim class="javascript">
+              let age = prompt("How old are you? Type a number");
+
+              console.log(age);           // ==> "30"
+              console.log(typeof age);    // ==> string
+
+              age = Number(age);     // converting to number
+
+              console.log(age);           // ==> 30
+              console.log(typeof age);    // ==> number
+
+              age = String(age);     // converting back to string
+
+              console.log(age);           // ==> "30"
+              console.log(typeof age);    // ==> string
+            </code></pre>
+          </section>
+        </section>
+
+        <section>
+          <section data-auto-animate data-autoslide="2000">
+            <p>Language Basics:</p>
+            <h2>Conditionals</h2>
+          </section>
+
+          <section data-auto-animate>
+            <h2>Conditionals</h2>
+
+            <p>Syntax:</p>
+
+            <pre><code data-trim class="javascript">
+              if ( /* condition */ ) {
+                /* code that runs when condition is true */
+              }
+            </code></pre>
+
+            <p style="font-size: .75em" class="fragment">The condition can be any expression which returns a boolean.</p>
+
+            <pre class="fragment"><code data-trim class="javascript">
+              if (true) {
+                console.log("I always run");
+              }
+
+              if (false) {
+                console.log("I never run");
+              }
+            </code></pre>
+
+            <aside class="notes">
+              <p>Refer back to implicit type conversion: condition is converted into boolean</p>
+            </aside>
+          </section>
+
+          <section id="conditionals-example-slide">
+            <h2>Conditionals</h2>
+
+            <pre><code data-trim class="javascript">
+              const passwordEntry = prompt("What is the password?");
+
+              if (passwordEntry === "hunter2") {
+                alert("You now have access to my super secret area!");
+              }
+            </code></pre>
+
+            <div style="display: inline-block">
+              <button type="button" class="run-button">Run</button>
+            </div>
+
+            <div class="additional-note fragment">
+              <p><code>===</code> is the strict equality operator. It returns true, if both arguments are matching types and values.</p>
+
+              <pre><code data-trim class="javascript">
+                console.log(5 === 5);      // ==> true
+                console.log(8 === 2 * 4);  // ==> true
+                console.log("5" === 5);    // ==> false
+              </code></pre>
+
+              <p>By contrast, the loose equality operator <code>==</code> only checks values, not types</p>
```

Comment:
Can we put it more as the loose equality allows for implicit type conversion before checking for the value?